client/my-sites/checkout/src/components/cost-overrides-list.tsx

```diff
@@ -400,11 +402,12 @@ function SingleProductAndCostOverridesList( { product }: { product: ResponseCart
 		}
 	);
 	const [ , streamlinedPriceExperimentAssignment ] = useStreamlinedPriceExperiment();
+	const monthlyPrices = useEquivalentMonthlyTotals( [ product ] );
 	if ( isStreamlinedPriceCheckoutTreatment( streamlinedPriceExperimentAssignment ) ) {
 		let streamlinedActualAmountDisplay;
 
-		// logic taken from packages/wpcom-checkout/src/checkout-line-items.tsx
-		const originalAmountInteger = product.item_original_subtotal_integer;
+		const originalAmountInteger =
+			monthlyPrices[ product.product_slug as PlanSlug ] || product.item_original_subtotal_integer;
```

Comment:
Just noting - Ideally, we shouldn't ever be working with snake_cased variables in JS/TS functions and components. These should have been transformed at some point earlier, or passed into a transformer/projection at a later point.